manifest_staging/charts/secrets-store-csi-driver/templates/_helpers.tpl

```diff
@@ -27,16 +27,27 @@ If release name contains chart name it will be used as a full name.
 {{/*
 Standard labels for helm resources
 */}}
-{{- define "sscd.labels" -}}
-labels:
-  app.kubernetes.io/instance: "{{ .Release.Name }}"
-  app.kubernetes.io/managed-by: "{{ .Release.Service }}"
-  app.kubernetes.io/name: "{{ template "sscd.name" . }}"
-  app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
-  app: {{ template "sscd.name" . }}
-  helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+{{- define "sscd.labels" }}
+app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+app.kubernetes.io/part-of: "{{ template "sscd.name" . }}"
+app.kubernetes.io/component: csi-driver
+app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+{{- include "sscd.selectorLabels" . }}
+helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+{{- if .Values.customLabels }}
+{{ toYaml .Values.customLabels }}
+{{- end }}
 {{- end -}}
 
+{{/*
+Selector labels
+*/}}
+{{- define "sscd.selectorLabels" }}
+app.kubernetes.io/name: {{ include "sscd.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app: {{ template "sscd.name" . }}
+{{- end }}
```

Comment:
Are these selectorLabels specific to any scenario? If so, perhaps we could get it from Values file and make it generic?

Reply:
Not really specific labels. They are standard labels (except `app`), created for example by the Helm CLI with the `helm create newchart` command.
Ex: https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/_helpers.tpl#L98